src/adapter/src/optimize/dataflows.rs

```diff
@@ -305,9 +304,13 @@ impl<'a> DataflowBuilder<'a> {
 
     /// Determine the given view's monotonicity.
     ///
-    /// This recursively traverses the expressions of all (materialized) views involved in the
+    /// This recursively traverses the expressions of all (non-materialized) views involved in the
```

Comment:
A similar comment change is needed in the function at the "(materialized) views".